src/testdrive/src/main.rs

```diff
@@ -119,7 +120,14 @@ async fn run(args: Args) -> Result<(), Error> {
                 );
                 (region, account.into(), credentials)
             }
-        };
+        }
+    };
+
+    if args.files.iter().any(|f| f.contains("esoteric")) {
+        // Esoteric tests include all AWS tests, where whether you are connecting to
+        // LocalStack or real AWS makes a big difference for debugging.
+        println!("testdrive is connecting to AWS_REGION={:?}", aws_region);
```

Comment:
I'd advocate for a format like:

```
Configuration parameters:
    AWS region: ...
    Kafka address: ...
    Schema registry URL: ...
    materialized URL: ....
```

I'd like to avoid saying things like "testdrive is connecting to..." here because testdrive is not actually connecting to anything at this moment, and indeed there are tests even in the esoteric directory that never touch AWS.

Reply:
This is the new shape of this PR.